web/src/app/core/user.service.ts

```diff
@@ -1,29 +1,52 @@
-import { Injectable } from "@angular/core";
+import { Injectable } from '@angular/core';
+import { HttpClient } from '@angular/common/http';
+import { HttpHeaders } from '@angular/common/http';
+import { Observable } from 'rxjs/observable';
+import 'rxjs/add/operator/map';
 import 'rxjs/add/operator/toPromise';
 import { AngularFirestore } from '@angular/fire/firestore';
 import { AngularFireAuth } from '@angular/fire/auth';
 import * as firebase from 'firebase/app';
+import { User } from 'firebase/app';
+import { FirebaseUserModel } from './user.model';
+
+const httpOptions = {
+  headers: new HttpHeaders({
+    'Content-Type':  'application/json',
+  })
+};
 
 @Injectable()
 export class UserService {
 
+  url:string = 'http://localhost:63073/api/user/' 
```

Comment:
You shouldn't hard-code the URL here. Instead you should use ``${environment.apiUrl}/user`` as the base URL will change depending on the environment the code is deployed on.

Reply:
I'm having some trouble when I do this, it says that it is unable to find 'src/environments/environment' even though I have imported it, same goes for the .prod and .stage.
Maybe I'm doing something wrong?